app/src/main/java/com/hedvig/app/feature/onboarding/ui/OnboardingAdapter.kt

```diff
@@ -116,16 +117,20 @@ class OnboardingAdapter(
 
             private fun animate(distance: Float) {
                 binding.apply {
-                    val shimmerStartPosition = shimmer.x
+                    val shimmerStartPosition = 0f - shimmer.width
                     Handler(Looper.getMainLooper()).postDelayed({
                         ValueAnimator.ofFloat(shimmerStartPosition, distance).apply {
                             duration = 1000
 
                             addUpdateListener { animation ->
                                 shimmer.translationX = animation.animatedValue as Float
                             }
+                            doOnStart {
+                                shimmer.visibility = View.VISIBLE
+                            }
                             doOnEnd {
                                 shimmer.translationX = shimmerStartPosition
+                                shimmer.visibility = View.INVISIBLE
```

Comment:
Should this be `GONE`?

Also, I recommend using the `core-ktx` extensions for manipulating visibility:

```suggestion
                                shimmer.isVisible = false
```